### ISSUE TITLE
Store unstranded feature locations correctly

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-chado#668]
We set a strand for every feature loaded into Chado to match the strand in the contig files.  That doesn't make sense for unstranded features like centromeres, repeats and low complexity regions.

@ValWood: is there any reason that a lot of unstranded features in the contig files have their location on the reverse strand.  Is it just an historical thing?  Here's an example:

    FT   repeat_region   complement(4298137..4298246)
    FT                   /systematic_id="SPRPTA.11"

Is there any reason to keep the strand information when we store these in Chado?

Needed for: https://github.com/pombase/website/issues/697

**Comment by ValWood** [comment]
I think the repeats are annotated so that they are all in the same orientation  a----b , that's why they are shown on specific strands.

For non repeat features they can be on any strand.

**Comment by kimrutherford** [comment]
The unstranded track is going to be quite empty as there aren't many non-repeat unstranded features.  Looks like less than 100.

**Comment by ValWood** [comment]
We should just put them on the forward strand, I reckon....

**Comment by kimrutherford** [comment]
> We should just put them on the forward strand, I reckon....

I've had a closer look and more of the features are stranded than I thought.

Are polyA_site and polyA_signal features stranded?

All the rest are misc_features with notes like:

"low complexity gene-free region"
"LTR-retrotransposon"
"previously annotated as SPAC1F12.03c"
"degraded Tf2 retrotransposable element"

It seems they need to be thought about case by case?

I think I'll close this issue and leave it as a curation task.  I won't change the loading code.

**Comment by ValWood** [comment]
> Are polyA_site and polyA_signal features stranded?

These are stranded.

**Comment by ValWood** [comment]
Reopened, as I don't know what to do here ( I think there are files for forward and reverse strand)

**Comment by kimrutherford** [comment]
> Reopened, as I don't know what to do here ( I think there are files for forward and reverse strand)

The features will be put in the forward or reverse strand GFF files based on their strand in Chado and in the contig files.  So if you want to move a feature to the other strand in the browser, edit the contig file and it should move to the other strand in JBrowse the next day.

**Comment by ValWood** [comment]
done @Antonialock  ?

**Comment by ValWood** [comment]
I don't think this is an issue, we are displaying the features in the strand they are annotated on is that correct? If so this can be closed. We can open if there is a further problem?

or are some features not displayed currently?

**Comment by kimrutherford** [comment]
> or are some features not displayed currently?

I think it's OK.  Perhaps we can just fix them as we see them?

**Comment by kimrutherford** [comment]
> we are displaying the features in the strand they are annotated on is that correct? 

Yep, that's right.  No processing is done at the moment.  We could easily (for example) show all repeat features on the forward strand if that made sense.
